src/profiles.jl

```diff
@@ -114,33 +126,26 @@ function Base.getindex(
     return _value_lookup(ScenarioIndexable(T), sp, period)
 end
 
-function ScenarioProfile(vals::Vector{Vector{T}}) where {T}
-    v = Vector{OperationalProfile{T}}()
-    for scv in vals
-        push!(v, OperationalProfile{T}(scv))
-    end
-    return ScenarioProfile(v)
-end
-
 """
-    RepresentativeProfile(vals)
+    RepresentativeProfile(vals::Vector{P}) where {T<:Duration, P<:TimeProfile{T}}
+    RepresentativeProfile(vals::Vector{<:Duration})
 
 Time profile with a separate time profile for each representative period.
 
-If too few profiles are provided, the last given profile will be
-repeated.
+If too few profiles are provided, the last given profile will be repeated.
+
+## Example
+```julia
+# Varying values in each strategic period
+profile = RepresentativeProfile([OperationalProfile([1, 2]), OperationalProfile([3, 4, 5])])
+ # The same value in each strategic period
```

Comment:
representative

Reply:
Copy paste error, to be updated for both `RepresentativeProfile` and `OperationalScenario`.